sympy/series/limits.py

```diff
@@ -136,6 +136,11 @@ def doit(self, **hints):
         if not e.has(z):
             return e
 
+        # limit can work with binomial by converting to factorial first
+        if (e.has(binomial) and not (hints.get('sequence', True)
```

Comment:
If I recall correctly, rewrite(tractable) is called only once in gruntz. If that rewrites binomial in terms of gamma, that will not be rewritten as tractable any more.